src/components/shared/layout/Sidebar.vue

```diff
@@ -144,4 +146,12 @@
             border-radius: 0;
         }
     }
+
+    @media (max-width: 768px) {
```

Comment:
Wouldn't it be 755px? as in `$collapsed-cc-menu-width` (55px) + `.ui.segments.universal-editor`'s width (700px). Either way, if it is a width specific to this component, it would be useful to use another sass variable here so it doesn't seem so arbitrary. 

```scss
$sidebar-max-width: 700px;
$collapsed-cc-menu-width: 55px;
$full-screen-breakpoint: $sidebar-max-width + $collapsed-cc-menu-width;
...
@media (max-width: $full-screen-breakpoint) {
...

```